roles/openshift_prometheus/README.md

```diff
@@ -23,6 +23,8 @@ For default values, see [`defaults/main.yaml`](defaults/main.yaml).
 
 - `openshift_prometheus_<COMPONENT>_image_version`: specify image version for the component 
 
+- `openshift_prometheus_<COMPONENT>_hostname`: specify the hostname for the route of that compontent: defaults to `<COMPONENT>-{{openshift_prometheus_namespace}}.{{openshift_master_default_subdomain}}`
+
```

Comment:
Would change the documentation to explicit `openshift_prometheus_hostname` and `openshift_prometheus_alerts_hostname`.
There are only 2 routes, and <COMPONENT> usually refers to: prometheus/alertmanager/alertbuffer/oauth-proxy